plugin-deps/src/main/java/android/content/LocalBroadcastManager.java

```diff
@@ -0,0 +1,10 @@
+package android.content;
+
+public class LocalBroadcastManager {
```

Comment:
I would add `android.support.v4.content.LocalBroadcastManager` type. It will be needed when an actual test on the class hierarchy is made. (See comment related to `InterfaceUtils.isSubtype()` below)